### ISSUE TITLE
new framework release (2.0.19) seems break the update function

### ISSUE THREAD

**Issue by samumist** [cgkineo/adapt-gulp-cli#8]
Got the following error message when running `adapt-setup update`  to update to the latest framework release (2.0.19). The update function worked fine previously.

```
No framework update found.
/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/node_modules/semver/semver.js:293
    throw new TypeError('Invalid Version: ' + version);
    ^

TypeError: Invalid Version: 1.0
    at new SemVer (/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/node_modules/semver/semver.js:293:11)
    at compare (/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/node_modules/semver/semver.js:566:10)
    at Function.lte (/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/node_modules/semver/semver.js:620:10)
    at Request._callback (/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/gulpfile.js:397:12)
    at Request.self.callback (/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/node_modules/request/request.js:186:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/Users/sam/.nvm/versions/node/v4.7.3/lib/node_modules/adapt-setup/node_modules/request/request.js:1081:10)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
```

**Comment by moloko** [comment]
@samumist thanks for flagging

@tomgreenfield could you have a look at this when you're back please?

**Comment by tomgreenfield** [comment]
Hi @samumist, it looks like you may have a dodgy `bower.json` in one of your plugins.

The error thrown states that `1.0` isn't a valid [semantic version](http://semver.org). To locate the offending plugin I'd suggest running:
```console
adapt-setup list
```

The `update` command only checks plugins in the registry so you must have a custom registry and/or modified plugin. Hope this makes sense.

**Comment by samumist** [comment]
Hi Tom
Sorry for the false alarm and thanks for the instruction.
Yes, I have a custom registry and I've found the offending plugin which has a version number of `1.0`. 

I was able to update other plugins after removing the offending plugin. However, the update couldn't find the new release of the framework (v2.0.19). Then I checked the [framework](https://github.com/adaptlearning/adapt_framework/blob/master/package.json#L3)'s `package.json` and its version is still `2.0.18`.

Many thanks for your help.
